packaging/nnstreamer.spec

```diff
@@ -195,9 +194,6 @@ BuildRequires: tensorflow-lite-devel
 %if 0%{?tensorflow2_lite_support}
 # for tensorflow2-lite
 BuildRequires: tensorflow2-lite-devel
-%if 0%{?tensorflow2_gpu_delegate_support}
```

Comment:
consider to change this requirements for the test
```
%if 0%{?tensorflow2_lite_support}
BuildRequires: pkgconfig(gles20)
%endif
```

Reply:
Currently, the filter does not support gl backend (only opencl backend for gpu delegate). I think gles20 dep is not required at this time.